<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Setup</class>
 <widget class="QWidget" name="Setup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1240</width>
    <height>281</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation Setup</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QPushButton" name="Exit">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>220</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="whatsThis">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exit reads the selections made in the setup panel and then closes the panel.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Finished</string>
   </property>
  </widget>
  <widget class="QLabel" name="Title">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>0</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>SimulatorSetup</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="SimLocation">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>550</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);
</string>
   </property>
   <property name="title">
    <string>Simulator Location</string>
   </property>
   <widget class="QLabel" name="Objectlbl">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>45</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Object</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Quadrant">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100,100,100);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Quadrantlbl">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>51</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Quadrant</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Sector">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100,100,100);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Object">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100,100,100);</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Location">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100,100,100);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Sectorlbl">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>70</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Sector</string>
    </property>
   </widget>
   <widget class="QLabel" name="Locationlbl">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>40</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Location</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SimCraft">
   <property name="geometry">
    <rect>
     <x>960</x>
     <y>20</y>
     <width>271</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="title">
    <string>SimCraft</string>
   </property>
   <widget class="QLabel" name="CraftNamelbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Craft Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="CraftClasslbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0,0,0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Craft Class</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CraftName">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100, 100, 100);
</string>
    </property>
   </widget>
   <widget class="QComboBox" name="CraftClass">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(100,100,100);
</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="SimType">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>271</width>
     <height>130</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgba(255, 255, 255, 0);</string>
   </property>
   <property name="title">
    <string>Simulator Type</string>
   </property>
   <widget class="QComboBox" name="Type">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>40</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="SimTypelbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Simulation Type</string>
    </property>
   </widget>
   <widget class="QLabel" name="Branchlbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Branch Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Branch">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>70</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
   <widget class="QLabel" name="Classlbl">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Class Type</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Class">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>100</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(100, 100, 100);
color: rgb(255, 255, 255);</string>
    </property>
   </widget>
  </widget>
  <widget class="QTextEdit" name="SimTypeInstruction">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>271</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Exit</sender>
   <signal>clicked()</signal>
   <receiver>Setup</receiver>
   <slot>Exit()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>30</x>
     <y>21</y>
    </hint>
    <hint type="destinationlabel">
     <x>511</x>
     <y>49</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Quadrant</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>QuadrantChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>290</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>460</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Location</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>LocationChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Sector</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>SectorChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>529</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Object</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>ObjectChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>540</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>44</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Type</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>TypeChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>260</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Branch</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>BranchChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>104</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Class</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>ClassChanged()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>209</x>
     <y>130</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>110</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CraftClass</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>CraftClass()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1129</x>
     <y>70</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>CraftName</sender>
   <signal>activated(QString)</signal>
   <receiver>Setup</receiver>
   <slot>CraftName()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1129</x>
     <y>100</y>
    </hint>
    <hint type="destinationlabel">
     <x>619</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>Exit()</slot>
  <slot>QuadrantChanged()</slot>
  <slot>RankChanged()</slot>
  <slot>SectorChanged()</slot>
  <slot>LocationChanged()</slot>
  <slot>ObjectChanged()</slot>
  <slot>TypeChanged()</slot>
  <slot>BranchChanged()</slot>
  <slot>ClassChanged()</slot>
  <slot>CraftClass()</slot>
  <slot>CraftName()</slot>
 </slots>
</ui>
